### ISSUE TITLE
@touchstart event ?

### ISSUE THREAD

**Issue by leRenart** [icebob/vue-touch-keyboard#29]
Hi, I'm using your (very helpful) plugin with a true kiosk machine that has no real keyboard input. It works really well, however we noticed that some elderly customers long-press on screen and as the whole plugin is based on click events, they feel the screen as unresponsive (they should leave the finger to see the input changed). It's a subtle feeling but it's there, and I wonder if a touchstart event could be better suited for this kind of interaction. 

Your thoughts ?

**Comment by icebob** [comment]
Hi,
I'm using it with different touch displays and the problem is that some touch driver generates only click event. So the better way is to handle both events and avoid duplicated events. Because from touch events the browser generates click event too.

**Comment by leRenart** [comment]
yeah, I see your point, will you accept a PR with this kind of change ?

https://codesandbox.io/s/lxq814kp0l

**Comment by icebob** [comment]
Yes, of course.

By the way, what is in the codesandbox link?

**Comment by leRenart** [comment]
Sorry I should have been more explicit:

    @touchstart="method"
    @click="method"

...

    method(e) {
      e.preventDefault();  // prevents triggering click event
    }

**Comment by icebob** [comment]
OK.

**Comment by andreasrein** [comment]
Any updates on this issue?
